Resources/Prototypes/SimpleStation14/Entities/Mobs/NPCs/primates.yml

```diff
@@ -0,0 +1,24 @@
+- type: entity
+  name: gorilla
+  parent: MobGorilla
+  suffix: Syndicate
+  id: MobGorillaSyndicate
+  description: Smashes, roars, looks cool. Don't stand near one.
+  components:
+  - type: Inventory
+    speciesId: syndimonkey
+    templateId: SyndicateMonkey
+  - type: InventorySlots
+  - type: Loadout
+    prototypes: [ SyndimonkeyGear ]
+  - type: SolutionRegeneration
+    solution: chemstream
+    generated:
+      reagents:
+        - ReagentId: Stimulants
+          Quantity: 0.20
+
+- type: startingGear
+  id: SyndimonkeyGear
+  equipment:
+    ears: ClothingHeadsetAltSyndicate
```

Comment:
This probably shouldn't be here? How do Upstreams handle it?

Reply:
ill fix it